app/assets/stylesheets/layout/_work_packages_full_view.sass

```diff
@@ -140,6 +142,17 @@ body.controller-work_packages.full-create
     display: block !important
     visibility: hidden
 
+.work-packages-full-view--resizer
+  position: fixed
+  .work-packages--resizer
+    height: 100vh
+    width: 18px
+    left: -2px
+    &::before
+      left: 0
+      // 130px is the size of header (55px) and toolbar with padding (75px)
+      top: calc((100vh - 130px)/2)
```

Comment:
This won't work since there is a breadcrumb which can be differently high. Also the toolbar can vary in the height depending on the screen size and the length of the title. As discussed please have a look at the implementations of the other resizers and adapt/merge the code.

Reply:
I adapted this calculation. Hopefully the new solution (with a sticky position) is now working for every screen.
Unfortunately, I couldn't find a solution that works for every resizer in our project, so I didn't see a way to sum up the styles for all resizer in a single .sass script.